src/routes/jira/jira-atlassian-connect-get.ts

```diff
@@ -1,18 +1,101 @@
 import { Request, Response } from "express";
-import { envVars }  from "config/env";
+import { envVars } from "config/env";
 import { EnvironmentEnum } from "interfaces/common";
+import { compact, map } from "lodash";
 
 const instance = envVars.INSTANCE_NAME;
 const isProd = (instance === EnvironmentEnum.production);
 // TODO: implement named routes (https://www.npmjs.com/package/named-routes) to facilitate rerouting between files
 export const postInstallUrl = "/jira/configuration";
-
 const key = `com.github.integration${instance ? `.${instance}` : ""}`;
-const conditions = [
+
+const adminCondition = [
 	{
 		condition: "user_is_admin"
 	}
 ];
+const modules = {
```

Comment:
This was abstracted from `JiraAtlassianConnectGet` so that we can dynamically set the `moduleUrls` below on line 98.